server/server.go

```diff
@@ -60,6 +61,6 @@ func runServer(config *Settings){
 	http.HandleFunc("/authenticate", func(w http.ResponseWriter, r *http.Request){
 		authenticate(w,r,config)
 	})
-	log.Fatal(http.ListenAndServe(":3000", nil))
+	log.Fatal(http.ListenAndServe(":" + strconv.Itoa(int(config.Port)), nil))
```

Comment:
I'll then make the whole listen address configurable? Or should I split it up in address and port? If you don't object I will go with the first.